src/domdiv/main.py

```diff
@@ -16,6 +16,14 @@
 from .cards import Card, CardType
 from .draw import DividerDrawer
 
+try:
+    from icu import Collator, Locale
+except ImportError:
```

Comment:
Could set a flag `have_icu=True` here to avoid having to check `sys.modules` later.

Reply:
Done, thanks for the suggestion